Reorganize the order of links to match the left menu items

bibryam opened the pull request:
Signed-off-by: Bilgin Ibryam <bibryam@gmail.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [+] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [+] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [+] Commands include options for Linux, MacOS, and Windows within codetabs
- [+] New file and folder names are globally unique
- [+] Page references use shortcodes instead of markdown or URL links
- [+] Images use HTML style and have alternative text
- [+] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Reorganize the order of links to match the left menu items

## Issue reference

<!--Please reference the issue this PR will close: #[issue number]-->

greenie-msft committed:
Update workflow and config.toml for v1.9

greenie-msft committed:
Merge pull request #2631 from greenie-msft/v1.9

Update workflow and config.toml for v1.9

berndverst committed:
Document SignalR and Storage Queues Binding AAD Support

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

akkie committed:
Add Zeebe autocomplete flag (#2673)

Signed-off-by: Christian Kaps <ck-github@mohiva.com>

hhunter-ms committed:
upmerge v1.8 to v1.9

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

artursouza committed:
Creating clarity on breaking changes and deprecations. (#2682)

Signed-off-by: Artur Souza <artursouza.ms@outlook.com>

cmendible committed:
Azure App Configuration docs

Signed-off-by: cmendible <cmendible@gmail.com>

hhunter-ms committed:
support policy grammar

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

hhunter-ms committed:
Merge branch 'v1.8' into upmerge_1.8_to_1.9

hhunter-ms committed:
Merge branch 'v1.9' into upmerge_1.8_to_1.9

addjuarez committed:
Update cassandra to Beta

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

addjuarez committed:
Update cassandra to Beta

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>

shubham1172 committed:
Update dapr publish CLI

Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

shubham1172 committed:
Update dapr publish CLI

Signed-off-by: Shubham Sharma <shubhash@microsoft.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

nyemade-uversky committed:
Patch 3 (#2691)

* Update release version from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* Update shortcode from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

1046102779 committed:
fix: typo

Signed-off-by: 1046102779 <seachen@tencent.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

greenie-msft committed:
Merge pull request #2684 from hhunter-ms/upmerge_1.8_to_1.9

Upmerge docs 1.8 to 1.9

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

greenie-msft committed:
Merge branch 'v1.9' into shubham1172/add-cli-publish-metadata-docs

greenie-msft committed:
Merge pull request #2689 from shubham1172/shubham1172/add-cli-publish-metadata-docs

Update dapr CLI publish documentation

greenie-msft committed:
Merge branch 'v1.9' into cassandraAlpha

addjuarez committed:
update to 1.9

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

addjuarez committed:
update to 1.9

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>

addjuarez committed:
Merge branch 'cassandraAlpha' of https://github.com/addjuarez/docs into cassandraAlpha

greenie-msft committed:
Merge pull request #2701 from addjuarez/cassandraAlpha

Promote cassandra to Beta

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

Co-authored-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/signalr.md

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Merge branch 'v1.9' into v1.8

greenie-msft committed:
Merge pull request #2652 from berndverst/v1.8

Document Storage Queue Binding and SignalR AAD Support

artursouza committed:
Change criteria for stable component (no stable candidate anymore).

Signed-off-by: Artur Souza <artursouza.ms@outlook.com>

greenie-msft committed:
Merge pull request #2687 from cmendible/issue-2660

Azure App Configuration configuration store docs

greenie-msft committed:
Merge branch 'v1.9' into update_component_cert

greenie-msft committed:
Merge pull request #2720 from artursouza/update_component_cert

Change criteria for stable component (no stable candidate anymore).

berndverst committed:
RethinkDB now Beta, but has no transactions, etags

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Update component status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Add components newly in beta

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Mark components as stable in v1.8

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
mark components as stable in v1.9

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

berndverst committed:
Azure Table Storage state is stable since v1.9

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

greenie-msft committed:
Merge pull request #2755 from berndverst/rethinkdb-v1.9

RethinkDB now Beta, but has no transactions, etags

greenie-msft committed:
Merge pull request #2758 from berndverst/statusv1.9

Marks components as beta and stable in the upcoming v1.9 release.

shubham1172 committed:
Upgrade JS-SDK submodule (#2767)

Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

yash-nisar committed:
Fix typos in MongoDB state configuration file example (#2776)

hhunter-ms committed:
Upmerge to 1.9 (#2793)

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Updates wasm middleware documentation with compilation instructions

These changes are coupled with https://github.com/dapr/components-contrib/pull/1833

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* Update daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>

* feedback

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* permute

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* add note to aws sns/sqs component doc; quick edit

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* change data to value

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Patch 3 (#2691)

* Update release version from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* Update shortcode from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* fix: typo

Signed-off-by: 1046102779 <seachen@tencent.com>

* feature: add a private helm repo for dapr

Signed-off-by: 1046102779 <seachen@tencent.com>

* Update install-dapr-cli.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Update daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>

* Fix pubsub quickstart component name to match QS source code

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* add output of success

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove word

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add sentence

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* redis 6 update (#2718)

Signed-off-by: yaron2 <schneider.yaron@live.com>

Signed-off-by: yaron2 <schneider.yaron@live.com>

* Update Dapr version to 1.8.4 (#2714)

* Update Dapr version to 1.8.4

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Update CLI version to 1.8.1

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>

* Fixed an issue with code highlighting

Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>

* Fix links to broken middleware components

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* RethinkDB has no transactions, etags

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* fix a typo (#2748)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* fix typo (#2751)

Signed-off-by: RcXu <honeyxhl@163.com>

Signed-off-by: RcXu <honeyxhl@163.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Update component status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Mark components as stable in v1.8

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Correct state azure stablestorage status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Update setup-sqlserver.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* fix some typos (#2765)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* Typo fix installtion > installation (#2726)

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Add a new version of the Dapr Diagrams deck (#2773)

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Fix typos in MongoDB state configuration file example (#2775)

Closes https://github.com/dapr/docs/issues/2774

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

* update service invo row item

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update az cli commands

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove unnecessary flags

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* remove ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: 1046102779 <seachen@tencent.com>
Signed-off-by: Adrian Cole <adrian@tetrate.io>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>
Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Signed-off-by: yaron2 <schneider.yaron@live.com>
Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Signed-off-by: Loong <loong.dai@intel.com>
Signed-off-by: RcXu <honeyxhl@163.com>
Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Signed-off-by: Yash Nisar <yashnisar@microsoft.com>
Co-authored-by: 1046102779 <seachen@tencent.com>
Co-authored-by: Adrian Cole <adrian@tetrate.io>
Co-authored-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Co-authored-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>
Co-authored-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Co-authored-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Co-authored-by: Loong Dai <loong.dai@intel.com>
Co-authored-by: Mr.Black <honeyxhl@163.com>
Co-authored-by: Greg. A <gautric@users.noreply.github.com>
Co-authored-by: Yash Nisar <yashnisar@microsoft.com>

amulyavarote committed:
Changed postgres output binding from Alpha to Stable

Signed-off-by: Amulya Varote <amulyavarote@microsoft.com>

greenie-msft committed:
Merge pull request #2794 from amulyavarote/postgres_binding_docs

Changed postgres output binding from Alpha to Stable

hhunter-ms committed:
Upmerge - 9/21 (#2818)

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Updates wasm middleware documentation with compilation instructions

These changes are coupled with https://github.com/dapr/components-contrib/pull/1833

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* Update daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>

* feedback

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* permute

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* add note to aws sns/sqs component doc; quick edit

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* change data to value

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Patch 3 (#2691)

* Update release version from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* Update shortcode from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* fix: typo

Signed-off-by: 1046102779 <seachen@tencent.com>

* feature: add a private helm repo for dapr

Signed-off-by: 1046102779 <seachen@tencent.com>

* Update install-dapr-cli.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Update daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>

* Fix pubsub quickstart component name to match QS source code

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* add output of success

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove word

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add sentence

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* redis 6 update (#2718)

Signed-off-by: yaron2 <schneider.yaron@live.com>

Signed-off-by: yaron2 <schneider.yaron@live.com>

* Update Dapr version to 1.8.4 (#2714)

* Update Dapr version to 1.8.4

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Update CLI version to 1.8.1

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>

* Fixed an issue with code highlighting

Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>

* Fix links to broken middleware components

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* RethinkDB has no transactions, etags

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* fix a typo (#2748)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* fix typo (#2751)

Signed-off-by: RcXu <honeyxhl@163.com>

Signed-off-by: RcXu <honeyxhl@163.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Update component status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Mark components as stable in v1.8

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Correct state azure stablestorage status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Update setup-sqlserver.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* fix some typos (#2765)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* Typo fix installtion > installation (#2726)

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Add a new version of the Dapr Diagrams deck (#2773)

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Fix typos in MongoDB state configuration file example (#2775)

Closes https://github.com/dapr/docs/issues/2774

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

* update service invo row item

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update az cli commands

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove unnecessary flags

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* remove ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* java submodule

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Fix two typo's (#2812)

Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>

Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>

Signed-off-by: 1046102779 <seachen@tencent.com>
Signed-off-by: Adrian Cole <adrian@tetrate.io>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>
Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Signed-off-by: yaron2 <schneider.yaron@live.com>
Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Signed-off-by: Loong <loong.dai@intel.com>
Signed-off-by: RcXu <honeyxhl@163.com>
Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Signed-off-by: Yash Nisar <yashnisar@microsoft.com>
Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>
Co-authored-by: 1046102779 <seachen@tencent.com>
Co-authored-by: Adrian Cole <adrian@tetrate.io>
Co-authored-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Co-authored-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>
Co-authored-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Co-authored-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Co-authored-by: Loong Dai <loong.dai@intel.com>
Co-authored-by: Mr.Black <honeyxhl@163.com>
Co-authored-by: Greg. A <gautric@users.noreply.github.com>
Co-authored-by: Yash Nisar <yashnisar@microsoft.com>
Co-authored-by: Maarten Mulders <mthmulders@users.noreply.github.com>

addjuarez committed:
cassanda state store cert (#2801)

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>

Signed-off-by: Addison Juarez <adjuarez@microsoft.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

pravinpushkar committed:
Updated azure app config metadata fields name (#2830)

* Updated azure app config metadata fields name

Signed-off-by: Pravin Pushkar <ppushkar@microsoft.com>

* Updating index for configuration store

Signed-off-by: Pravin Pushkar <ppushkar@microsoft.com>

Signed-off-by: Pravin Pushkar <ppushkar@microsoft.com>

akhilac1 committed:
postgres configuration store (#2800)

* postgres configuration store

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>

* Added example for subscribe request

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>

* added pool_max_conns to example connection string

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>

* added recommendation to use single subscription with multiple keys

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>

* fixed review comments

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>

Signed-off-by: akhilac1 <chetlapalle.akhila@gmail.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

bibryam committed:
Reorganize the order of links to match the left menu items

Signed-off-by: Bilgin Ibryam <bibryam@gmail.com>

hhunter-ms reviewed (approved):
@bibryam pulling latest from v1.8 should fix the link validation error

hhunter-ms committed:
Upmerge v1.8 --> v1.9 - 9/28 (#2839)

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Updates wasm middleware documentation with compilation instructions

These changes are coupled with https://github.com/dapr/components-contrib/pull/1833

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* Update daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

Co-authored-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>

* feedback

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* permute

Signed-off-by: Adrian Cole <adrian@tetrate.io>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* add note to aws sns/sqs component doc; quick edit

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* change data to value

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* feature/helmchart: supported private helm chart repo to install dapr cluster

Signed-off-by: 1046102779 <seachen@tencent.com>

* Patch 3 (#2691)

* Update release version from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* Update shortcode from 1.8.2 to 1.8.3

Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>

* fix: typo

Signed-off-by: 1046102779 <seachen@tencent.com>

* feature: add a private helm repo for dapr

Signed-off-by: 1046102779 <seachen@tencent.com>

* Update install-dapr-cli.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Update daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>

* Fix pubsub quickstart component name to match QS source code

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* add output of success

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove word

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add sentence

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* redis 6 update (#2718)

Signed-off-by: yaron2 <schneider.yaron@live.com>

Signed-off-by: yaron2 <schneider.yaron@live.com>

* Update Dapr version to 1.8.4 (#2714)

* Update Dapr version to 1.8.4

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Update CLI version to 1.8.1

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>

* Fixed an issue with code highlighting

Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>

* Fix links to broken middleware components

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* RethinkDB has no transactions, etags

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* fix a typo (#2748)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* fix typo (#2751)

Signed-off-by: RcXu <honeyxhl@163.com>

Signed-off-by: RcXu <honeyxhl@163.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Update component status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Mark components as stable in v1.8

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Correct state azure stablestorage status in v1.7

Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

* Update setup-sqlserver.md

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* fix some typos (#2765)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* Typo fix installtion > installation (#2726)

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>

Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Add a new version of the Dapr Diagrams deck (#2773)

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

* Fix typos in MongoDB state configuration file example (#2775)

Closes https://github.com/dapr/docs/issues/2774

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

* update service invo row item

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update az cli commands

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove unnecessary flags

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* remove ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* java submodule

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Fix two typo's (#2812)

Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>

Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>

* Updates/fixes to mySQL binding doc (#2819)

* update link (#2823)

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* fix the example code (#2811)

Signed-off-by: Loong <loong.dai@intel.com>

Signed-off-by: Loong <loong.dai@intel.com>

* fix incorrect reference (#2802)

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Fix Configuration API unsubscribe endpoint  (#2797)

* fix unsubscribe path

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

* maintain punctuation uniformity

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

* maintain punctuation uniformity

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

* Update configuration_api.md

Making examples consistent

Signed-off-by: Mark Fussell <markfussell@gmail.com>

Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* fix link

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* Update go commands to match quickstarts (#2725)

* update go commands

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>

* Add note for sidecar readiness (#2817)

* Revert "java submodule"

This reverts commit b438550fbb62b4f429f4728a45147209fe8f5d0a.

* revert submodule changes

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove note

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update link

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Docs contributing guide updates (#2768)

* initial edits

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* updates

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add templates

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* fix links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove path

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add docs

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove from .github doc

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* fix validate links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* updates per mark

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* add submodule section

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* remove link and reword

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* fix link?

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* apparently case matters for links now

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update per mark review

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

* update link

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* Changed the broken middleware links. (#2781)

* Add a new version of the Dapr Diagrams deck (#2773)

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Fix typos in MongoDB state configuration file example (#2775)

Closes https://github.com/dapr/docs/issues/2774

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>

Signed-off-by: Yash Nisar <yashnisar@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Changed the broken middleware links.

Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Made the required changes for Rego/OPA

Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* update az cli commands

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* remove unnecessary flags

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* remove ignore links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Fix broken link

Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* update service invo row item

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

* Fixed Rego/OPA broken link

Signed-off-by: Shivani Singh <shivasingh@microsoft.com>

Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>
Signed-off-by: Yash Nisar <yashnisar@microsoft.com>
Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Co-authored-by: Yash Nisar <yashnisar@microsoft.com>
Co-authored-by: Hannah Hunter <hannahhunter@microsoft.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>

* fix links

Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

Signed-off-by: 1046102779 <seachen@tencent.com>
Signed-off-by: Adrian Cole <adrian@tetrate.io>
Signed-off-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>
Signed-off-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Signed-off-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Signed-off-by: yellow chicks <seachen@tencent.com>
Signed-off-by: Nick Greenfield <nigreenf@microsoft.com>
Signed-off-by: yaron2 <schneider.yaron@live.com>
Signed-off-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Signed-off-by: Loong <loong.dai@intel.com>
Signed-off-by: RcXu <honeyxhl@163.com>
Signed-off-by: Greg. A. <gautric@users.noreply.github.com>
Signed-off-by: Yash Nisar <yashnisar@microsoft.com>
Signed-off-by: Maarten Mulders <mthmulders@users.noreply.github.com>
Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>
Signed-off-by: Mark Fussell <markfussell@gmail.com>
Signed-off-by: Shivani Singh <shivasingh@microsoft.com>
Co-authored-by: 1046102779 <seachen@tencent.com>
Co-authored-by: Adrian Cole <adrian@tetrate.io>
Co-authored-by: Crypt Keeper <64215+codefromthecrypt@users.noreply.github.com>
Co-authored-by: Mark Fussell <markfussell@gmail.com>
Co-authored-by: greenie-msft <56556602+greenie-msft@users.noreply.github.com>
Co-authored-by: nyemade-uversky <55847877+nyemade-uversky@users.noreply.github.com>
Co-authored-by: Yaron Schneider <schneider.yaron@live.com>
Co-authored-by: Nick Greenfield <nigreenf@microsoft.com>
Co-authored-by: Lorenzo Montanari <l0ll098@users.noreply.github.com>
Co-authored-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>
Co-authored-by: Loong Dai <loong.dai@intel.com>
Co-authored-by: Mr.Black <honeyxhl@163.com>
Co-authored-by: Greg. A <gautric@users.noreply.github.com>
Co-authored-by: Yash Nisar <yashnisar@microsoft.com>
Co-authored-by: Maarten Mulders <mthmulders@users.noreply.github.com>
Co-authored-by: Boris Lublinsky <blublinsky@hotmail.com>
Co-authored-by: Sarthak Sharma <sartsharma@microsoft.com>
Co-authored-by: shivanisinghnitp <58537821+shivanisinghnitp@users.noreply.github.com>

msfussell committed:
Merge branch 'v1.9' into quickstarts-links

hhunter-ms commented:
@bibryam could you point this PR to v1.8 instead of v1.9?

bibryam commented:
Sorry for the mess, I'll close this and make a new PR with the right base version.
